examples/offline_inference_neuron.py

```diff
@@ -32,7 +32,9 @@
     # The device argument can be either unspecified for automated detection,
     # or explicitly assigned.
     device="neuron",
-    tensor_parallel_size=2)
+    tensor_parallel_size=2,
+    # Single node inference
+    world_size=1)
```

Comment:
i think the consistency of user interface matters. it's hard to understand what to expect when we set default as TP=2, PP=1, world_size=1

Reply:
Done, refactored to keep interface consistent.